examples/tutorial/tutorial_part_two.ipynb

```diff
@@ -252,7 +252,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "clipper.cleanup()"
+    "clipper.stop_all()"
```

Comment:
By making this replacement, we've rendered our "Warning" image and description irrelevant as `cleanup()` is no longer called throughout the course of the tutorial. We should probably remove the warning info.